src/pages/Swap/SimpleSwap.tsx

```diff
@@ -0,0 +1,217 @@
+import './SimpleSwap.css'
+
+import { Box, Flex, Grid, GridItem } from '@chakra-ui/layout'
+import { Switch, useMediaQuery } from '@chakra-ui/react'
+import { useState } from 'react'
+import { Page } from 'components/Layout/Page'
+import { breakpoints } from 'theme/theme'
+
+import { CenterColumn } from './components/CenterColumn'
+import { CenterRow } from './components/CenterRow'
+import { ClassicVersionTradeSubmitted } from './components/ClassicVersionTradeSubmitted'
+import { MobileFooter } from './components/MobileFooter'
+import { OGHeader } from './components/OGHeader'
+import { SwapCardSwitch } from './components/SwapCardSwitch'
+import {
+  BODY_PADDING_TOP,
+  DEFAULT_BUY_ASSET,
+  DEFAULT_SELL_ASSET,
+  OG_COLORS,
+  PAGE_HEIGHT_OFFSET_CLASSIC,
+  PAGE_HEIGHT_OFFSET_MOBILE,
+  PAGE_HEIGHT_OFFSET_MODERN,
+  PAGE_MARGIN_TOP_MODERN,
+  TRADE_STEPS_CLASSIC,
+  TRADE_STEPS_MODERN,
+} from './constants'
+
+export const SimpleSwap = () => {
+  const [shouldShowClassicVersion, setShouldShowClassicVerion] = useState(false)
+  const [currentStepIndex, setCurrentStepIndex] = useState(0)
+  const [sellAsset, setSellAsset] = useState(DEFAULT_SELL_ASSET)
+  const [buyAsset, setBuyAsset] = useState(DEFAULT_BUY_ASSET)
+
+  const stepsList = shouldShowClassicVersion ? TRADE_STEPS_CLASSIC : TRADE_STEPS_MODERN
+  const currentStep = stepsList[currentStepIndex]
+  const bgColor = shouldShowClassicVersion ? OG_COLORS.lightBlue : OG_COLORS.darkBlue
+  const hasSubmittedTrade = currentStep === 'processing' || currentStep === 'complete'
+  const isInitialStep = currentStep === 'initial'
+  const isFinalStep = currentStep === 'complete'
```

Comment:
(minor) I have gotten into the practice of using `useMemo` in replace of 'non-scoped' variables. I found it improves performance of a component loading. But this is super nic-picking so not required to change

Reply:
@Da-Colon I'm not sure if I understand what use case you're thinking of—could you give an example? Are you suggesting converting `isFinalStep` to a function, for instance, and passing it to `useMemo()`?

Reply:
You can ignore this comment. I have since come to a better understanding of `use3Memo` and what I was suggesting was overkill for this situation. @cmccand